src/Controls/src/Core/Handlers/NavigationPage/NavigationPageHandler.cs

```diff
@@ -14,9 +14,9 @@ public partial class NavigationPageHandler
 		public static PropertyMapper<NavigationPage, NavigationPageHandler> NavigationPageMapper
 			   = new PropertyMapper<NavigationPage, NavigationPageHandler>(ViewHandler.ViewMapper)
 			   {
-				   [NavigationPage.BarTextColorProperty.PropertyName] = MapTitleColor,
-				   [NavigationPage.BarBackgroundColorProperty.PropertyName] = MapNavigationBarBackground,
-				   [NavigationPage.BarBackgroundProperty.PropertyName] = MapNavigationBarBackground,
+				   [NavigationPage.BarTextColorProperty.PropertyName] = MapBarTextColor,
+				   [NavigationPage.BarBackgroundColorProperty.PropertyName] = MapBarBackground,
+				   [NavigationPage.BarBackgroundProperty.PropertyName] = MapBarBackground,
```

Comment:
`BarBackground` is a Brush, and in .NET MAUI we will use Maui.Graphics Paint objects. For now, could we remove all the brushes code (extensions etc) and leave the colors for now?

Reply:
All of this code is inside Controls.Core not inside Maui.Core

So the brushes that are being used here are the bindable ones.

Also, the plan is to move most of this code back to Maui.Core at which point this will all be based on paint.